assets/blocks/quiz/quiz-block/quiz-settings.js

```diff
@@ -86,16 +90,72 @@ const QuizSettings = ( {
 					/>
 				</PanelRow>
 				{ passRequired && (
-					<PanelRow>
-						<RangeControl
-							label={ 'Passing Grade (%)' }
-							value={ quizPassmark }
-							onChange={ createChangeHandler( 'quizPassmark' ) }
-							min={ 0 }
-							max={ 100 }
-							initialPosition={ 100 }
-						/>
-					</PanelRow>
+					<>
+						<PanelRow>
+							<RangeControl
+								label={ 'Passing Grade (%)' }
+								value={ quizPassmark }
+								onChange={ createChangeHandler(
+									'quizPassmark'
+								) }
+								min={ 0 }
+								max={ 100 }
+								initialPosition={ 100 }
+							/>
+						</PanelRow>
+						<PanelRow>
+							<div>
+								<BaseControl
+									id={
+										'sensei-lms-quiz-block-failed-feedback-options'
```

Comment:
Not critical, but it could be `id="sensei-lms-quiz-block-failed-feedback-options"`